site/content/en/topics/assumptions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Assumptions for Provisioning Tools"
+linkTitle: "Assumptions for Provisioning Tools"
+weight: 1
+type: docs
+description: >
+    Cloud provider assumptions on Azure resources that provisioning tools should take care.
+---
+
+Here is a list of Azure resource assumptions that are required for cloud provider Azure:
+
+* All Azure resources are under the same tenant.
+* All the virtual machines are part of either VirtualMachineScaleSet (VMSS) or AvailabilitySet (VMAS).
+* All virtual machine's name should be the same as their hostname.
```

Comment:
does it apply for Windows? cc @jsturtevant

Reply:
yes